src/stores/queueStore.ts

```diff
@@ -253,14 +260,16 @@ interface State {
   pendingTasks: TaskItemImpl[]
   historyTasks: TaskItemImpl[]
   maxHistoryItems: number
+  loadingHistory: boolean
 }
 
 export const useQueueStore = defineStore('queue', {
   state: (): State => ({
     runningTasks: [],
     pendingTasks: [],
     historyTasks: [],
-    maxHistoryItems: 64
+    maxHistoryItems: 64,
+    loadingHistory: false
```

Comment:
Rename the variable to `isLoading`? `loadingHistory` can be a bit confusing. (A field that records the loading history of the queue?)